src/turtle.rs

```diff
@@ -199,6 +199,46 @@ impl Turtle {
         block_on(self.turtle.wait(secs))
     }
 
+    /// Instructs the turtle to draw an arc by the given angle for a given radius in the given step count from the current position of the turtle.
```

Comment:
```suggestion
    /// Draws a circle or an arc with the given radius from the current position of the turtle.
```

Updated based on the docs for the [circle](https://docs.python.org/3.3/library/turtle.html?highlight=turtle#turtle.circle) function in the Python turtle module